### DISCUSSION TITLE
Insufficient token scopes

### DISCUSSION THREAD

**Discussion by mateusfg7** [lowlighter/metrics#1553] [Q&A] [answered]
I've been facing this error for a while now, I have no idea what it is. I've already tried using `lowlighter/metrics@96dde98a034ab75fec7884f9161bc12759275d36`, I've already tried using `lowlighter/metrics@latest`, and `lowlighter/metrics@master`. All with the same problem.

![Anotação 2023-10-19 212150](https://github.com/lowlighter/metrics/assets/40613276/ecc01835-9d24-4636-8866-f0746588ae1f)

```graphql
[
  {
    name: 'traffic',
    result: { error: { message: 'Insufficient token scopes' } }
  }
]
```

![image](https://github.com/lowlighter/metrics/assets/40613276/e2f34774-1835-4b24-ab80-387650bf594b)

[my `metrics.yml`](https://github.com/mateusfg7/mateusfg7/blob/master/.github/workflows/metrics.yml)

**Comment by Vault108** [comment]
Check the permissions on the token in your repo secrets.

**Reply by mateusfg7** [reply]
I already have all necessary permissions

![image](https://github.com/lowlighter/metrics/assets/40613276/7175423d-bfe0-4736-b374-6534a80e2ac1)

**Comment by Vault108** [comment]
Per the read me it shows that when it is missing a scope.:  ![image](https://github.com/lowlighter/metrics/assets/44564111/eb3e0bb3-62c2-4ac6-b6f5-37989f06bcf2)
I would make a scopeless token as per https://github.com/lowlighter/metrics/blob/master/.github/readme/partials/documentation/setup/action.md#1%EF%B8%8F-create-a-github-personal-token

**Reply by mateusfg7** [reply]
I tried, dint'n work. 😢 

![image](https://github.com/lowlighter/metrics/assets/40613276/c2b20986-0640-4ef3-a9e5-cefce2d8e8e9)

![image](https://github.com/lowlighter/metrics/assets/40613276/34c06cc7-254c-4804-a379-a36dff18af01)

https://github.com/mateusfg7/mateusfg7/commit/5faf3f3614e27718e62137f14e70830f3ff3d7e9

**Comment by mateusfg7** [comment, marked as answer]
I was able to resolve the issue by changing the `repositories: 150` setting to `repositories: 78`. Apparently the problem was the Github API rate limit. But I'm going to keep issue #1558 open because I think the error returned doesn't match the apparent problem.
